finanzas.gob.ec,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
finda.co.nz,"HTTPSConnectionPool(host='finda.co.nz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='finda.co.nz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
fing.edu.uy,"HTTPSConnectionPool(host='fing.edu.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6ce006e820>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='fing.edu.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6ce10c1a30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
finnomena.com,"HTTPSConnectionPool(host='www.finnomena.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.finnomena.com', port=443): Read timed out. (read timeout=2)"
firat.edu.tr,"HTTPSConnectionPool(host='firat.edu.tr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cdf4e9640>, 'Connection to firat.edu.tr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='firat.edu.tr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f6cdfc65760>, 'Connection to firat.edu.tr timed out. (connect timeout=2)'))"
firmasec.com,"HTTPSConnectionPool(host='www.firmasec.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.firmasec.com', port=443): Read timed out. (read timeout=2)"
firstbanknigeria.com,"HTTPSConnectionPool(host='firstbanknigeria.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cde9995b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='firstbanknigeria.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6ce06641c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
fiscathema.nl,"HTTPSConnectionPool(host='fiscathema.nl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cde70cee0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='fiscathema.nl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6ce15c25b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
fisheries.go.th,"HTTPSConnectionPool(host='fisheries.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cdec98d30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='fisheries.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6cdf0adb80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
flagma.ru,"HTTPSConnectionPool(host='flagma.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='flagma.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
flagma.ua,"HTTPSConnectionPool(host='flagma.ua', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='flagma.ua', port=443): Max retries exceeded with url: /uk/ (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
flashback.org,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
flydubai.com,"HTTPSConnectionPool(host='flydubai.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='flydubai.com', port=80): Read timed out. (read timeout=2)"
flytap.com,"HTTPSConnectionPool(host='www.flytap.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.flytap.com', port=443): Read timed out. (read timeout=2)"
fmed.edu.uy,"HTTPSConnectionPool(host='fmed.edu.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cdd794a00>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='fmed.edu.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6ce1241220>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
fnbrno.cz,"HTTPSConnectionPool(host='fnbrno.cz', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cdd935070>, 'Connection to fnbrno.cz timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.fnbrno.cz', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ce0628fa0>, 'Connection to www.fnbrno.cz timed out. (connect timeout=2)'))"
focloir.ie,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
folketingstidende.dk,"HTTPSConnectionPool(host='folketingstidende.dk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cdcff9910>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPSConnectionPool(host='www.folketingstidende.dk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
fonasa.cl,"HTTPSConnectionPool(host='fonasa.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='fonasa.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
food-of-dream.com,"HTTPSConnectionPool(host='food-of-dream.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='food-of-dream.com', port=443): Read timed out. (read timeout=2)"
food4less.com,"HTTPSConnectionPool(host='food4less.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cdf4d3a90>, 'Connection to food4less.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.food4less.com', port=80): Read timed out. (read timeout=2)"
foodfolder.se,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='www.foodfolder.se', port=80): Read timed out. (read timeout=2)"
foodnetworksolution.com,"HTTPSConnectionPool(host='foodnetworksolution.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='foodnetworksolution.com', port=80): Read timed out. (read timeout=2)"
footshop.cz,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
form-zenoma.jp,"HTTPSConnectionPool(host='form-zenoma.jp', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='form-zenoma.jp', port=80): Read timed out. (read timeout=2)"
forrent.com,"HTTPSConnectionPool(host='forrent.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='forrent.com', port=80): Read timed out. (read timeout=2)"
forumdacasa.com,"HTTPSConnectionPool(host='forumdacasa.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='forumdacasa.com', port=443): Read timed out. (read timeout=2)"
fotorecept.com,"HTTPSConnectionPool(host='fotorecept.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cdf027a30>, 'Connection to fotorecept.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='fotorecept.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f6ce0a09850>, 'Connection to fotorecept.com timed out. (connect timeout=2)'))"
frag-mutti.de,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
fredmeyer.com,"HTTPSConnectionPool(host='fredmeyer.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cdac79ac0>, 'Connection to fredmeyer.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.fredmeyer.com', port=80): Read timed out. (read timeout=2)"
freundferreteria.com,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
frysfood.com,"HTTPSConnectionPool(host='frysfood.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cd9e0ed60>, 'Connection to frysfood.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.frysfood.com', port=80): Read timed out. (read timeout=2)"
ft.dk,"HTTPSConnectionPool(host='ft.dk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='ft.dk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
fugleognatur.dk,"HTTPSConnectionPool(host='www.fugleognatur.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.fugleognatur.dk', port=443): Read timed out. (read timeout=2)"
fujisan.co.jp,"HTTPSConnectionPool(host='fujisan.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.fujisan.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
fukui.lg.jp,"HTTPSConnectionPool(host='fukui.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cdca186a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='fukui.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6cdd7d3a00>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
fukuoka.lg.jp,"HTTPSConnectionPool(host='fukuoka.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cd9f9f670>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='fukuoka.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6cd8db7d90>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
fullgaz.co.il,"HTTPSConnectionPool(host='fullgaz.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='fullgaz.co.il', port=80): Read timed out. (read timeout=2)"
fundsupermart.com,"HTTPSConnectionPool(host='fundsupermart.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cdf102070>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='fundsupermart.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6cdf5698e0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
fyysika.ee,"HTTPSConnectionPool(host='www.fyysika.ee', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.fyysika.ee', port=80): Read timed out. (read timeout=2)"
g-pra.com,"HTTPSConnectionPool(host='g-pra.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cdec7b2e0>, 'Connection to g-pra.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='g-pra.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f6cdd71d2e0>, 'Connection to g-pra.com timed out. (connect timeout=2)'))"
g3grouplimited.com,"HTTPSConnectionPool(host='g3grouplimited.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='g3grouplimited.com', port=80): Read timed out. (read timeout=2)"
gabler.de,"HTTPSConnectionPool(host='gabler.de', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gabler.de' doesn't match either of 'gabler-banklexikon.de', 'gablerbanklexikon.de', 'm.wirtschaftslexikon.gabler.de', 'springergabler-bankinglexicon.com', 'wirtschaftslexikon.gabler.de', 'www.gabler-banklexikon.de', 'www.gablerbanklexikon.de', 'www.springergabler-bankinglexicon.com', 'www.wirtschaftslexikon.gabler.de'"")))","HTTPSConnectionPool(host='www.springer.com', port=443): Read timed out. (read timeout=2)"
gaccom.jp,"HTTPSConnectionPool(host='gaccom.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6cdaffab80>, 'Connection to gaccom.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='gaccom.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f6ce134f910>, 'Connection to gaccom.jp timed out. (connect timeout=2)'))"
gadgets-africa.com,"HTTPSConnectionPool(host='gadgets-africa.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='gadgets-africa.com', port=443): Read timed out. (read timeout=2)"
gadgetshop.co.il,"HTTPSConnectionPool(host='gadgetshop.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='gadgetshop.co.il', port=443): Read timed out. (read timeout=2)"
galamart.ru,"HTTPSConnectionPool(host='galamart.ru', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='galamart.ru', port=443): Read timed out. (read timeout=2)"
gamerant.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gamestop.com,"HTTPSConnectionPool(host='www.gamestop.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.gamestop.com', port=80): Read timed out. (read timeout=2)"
gantep.edu.tr,"HTTPSConnectionPool(host='gantep.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.gantep.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
garantibbva.com.tr,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='garantibbva.com.tr', port=80): Read timed out. (read timeout=2)"
garsigalatvija.lv,"HTTPSConnectionPool(host='garsigalatvija.lv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cdb011580>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.garsigalatvija.lv', port=80): Read timed out. (read timeout=2)"
gazetadeagricultura.info,"HTTPSConnectionPool(host='gazetadeagricultura.info', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gazetadeagricultura.info', port=443): Read timed out. (read timeout=2)"
gazzettaufficiale.it,"HTTPSConnectionPool(host='gazzettaufficiale.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cd86b0160>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='gazzettaufficiale.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6cdf891040>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
gcvmproductions.com,"HTTPSConnectionPool(host='gcvmproductions.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6ce00efc70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gcvmproductions.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6cde81f160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gd.se,"HTTPSConnectionPool(host='gd.se', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.gd.se', port=443): Read timed out. (read timeout=2)"
geeksforgeeks.org,"HTTPSConnectionPool(host='geeksforgeeks.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6cdbb9c310>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='geeksforgeeks.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6ce1b771c0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
gelbe-liste.de,"HTTPSConnectionPool(host='www.gelbe-liste.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gelbe-liste.de', port=443): Read timed out. (read timeout=2)"
gemoss.lv,"HTTPSConnectionPool(host='gemoss.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.gemoss.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
genghis-capital.com,"HTTPSConnectionPool(host='genghis-capital.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'genghis-capital.com' doesn't match 'www.genghis-capital.com'"")))","HTTPSConnectionPool(host='genghis-capital.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'genghis-capital.com' doesn't match 'www.genghis-capital.com'"")))"
